core/src/main/scala/kafka/tools/ExportZkOffsets.scala

```diff
@@ -54,26 +54,25 @@ object ExportZkOffsets extends Logging {
                             .withRequiredArg()
                             .defaultsTo("localhost:2181")
```

Comment:
If I don't provide the `--zkconnect` option, I get `Missing required option(s) [zkconnect]`. So, `defaultsTo` doesn't seem to work. I think it should either be removed or fixed (if you think this fits under this PR). This seems to work fine for `ImportZkOffsets`. Also, if we get `defaultsTo` working, this option should no longer be required.